frameworks/Swift/vapor/setup-mongodb.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+fw_depends swift3 mongodb mysql postgresql
+
+sudo apt-get install -y libmysqlclient-dev
```

Comment:
You actually should be able to include any tools that are installed with `apt-get` by including the package name in your `fw_depends` line. See [here](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/toolset/setup/linux/bash_functions.sh#L118) for more info. Basically, if fw_depends doesn't find a file with the given name in `toolset/setup/linux/`, it will attempt to install it with `apt-get`.